controls/basics/how-to-git.md

```diff
@@ -1,12 +1,33 @@
-## How to use git
+## What is git and what is it used for? 
 
-### What is git
+<img src="../../assets/images/git-logo.png" alt="git logo" style="width:150px; float: right; padding: 25px;">
+Git is something called a content tracker which is capable of cataloging all changes you make to tracked files. Git centers around the manipulation of branches of code, the merging and converging of existing code branches to improve a ending product. When you create or edit a file on your computer, the changes you make locally will be recorded and can differ from that of the global codebase otherwise known as the main branch. This dissonance provides greater quality control to the maintainer (permitting these changes), and allows for a more adaptable development cycle for the programmer through quality of life commands such as: "git revert", "git diff", and "git merge" How all of these benefits are achieved is through proper utilization.
```

Comment:
```suggestion
Git is a version control system which is capable of cataloging all changes you make to tracked files. Git centers around the manipulation of branches of code, to work separately on and eventually merge different sets of code to collaborate on a final product. When you create or edit a file on your computer, the changes you make locally can be recorded and can differ from that of the "global" code repository otherwise known as `origin` (for us, this one is hosted under the frc1675 organization). We use git with a specific set of commands and GitHub interactions to (mostly) enforce a process that provides a higher level of quality assurance via review and testing before new code joins the rest. How all of these benefits are achieved is through proper utilization.
```